application/src/ext-test/java/org/opentripplanner/ext/vehicleparking/bikely/BikelyUpdaterTest.java

```diff
@@ -8,7 +8,7 @@
 import java.time.Duration;
 import java.util.Locale;
 import org.junit.jupiter.api.Test;
-import org.opentripplanner.routing.vehicle_parking.VehicleParkingState;
+import org.opentripplanner.service.vehicleparking.model.VehicleParkingState;
```

Comment:
I feel like the package structure is slightly wrong maybe. At least for transit stuff, the structure is reversed, transit.service, not service.transit. Also, should the model be then vehicleparking.model while service in in vehicleparking.service?

Reply:
It follows the guidelines. See above ^^.